scripts/workers/worker_util.py

```diff
@@ -92,7 +93,7 @@ def update_submission_data(self, data, challenge_pk, submission_pk):
         return response
 
     def update_submission_status(self, data, challenge_pk):
-        url = "/api/jobs/challenge/{}/update_submission/".format(challenge_pk)
+        url = URLS.get("update_submission_status").format(challenge_pk)
```

Comment:
`update_submission_status` -> `update_submission_data`

Reply:
How about deleting `update_submission_status` and changing `update_submission_data` to `update_submission`?

Reply:
@vkartik97 Since `update_submission_data` will be used by both `*_data` and `*_status`, I think the change will be reasonable.